README.md

```diff
@@ -109,7 +109,18 @@ List of external core examples adopted to the latest framework:
 - [COMET-CHAM from Virginia Tech](https://github.com/kammoh/comet_cham_lwc_v2/tree/asic): VHDL (`asic` branch)
 - [SpoC from Virginia Tech](https://github.com/kammoh/comet_cham_lwc_v2/tree/asic): Verilog (`asic` branch)
 
-
+## Popular Configurable LWC Options
```

Comment:
Could we think of a better wording instead of "popular"? Notable? Primary?

Reply:
fixed